### PR TITLE
Fix local server running

### PR DESCRIPTION
**Pull request by freevova**

### CODE REVIEW

**Comment by freevova** [comment]
@EnragedWildkin Please review this PR.

**Comment by mir4a** [comment]
@EnragedWildkin do not merge this, the problem with foundation stylesheets import is not resolved

**Comment by EnragedWildkin** [comment]
@mir4a What should be done to resolve issues with stylesheets?

**Comment by mir4a** [comment]
@EnragedWildkin gulp should regenerate changed stylesheets without breaking site

**Comment by freevova** [comment]
@EnragedWildkin I added full path in app.scss. Now gulp regenerate changed stylesheets without breaking site. Please review.

File: _sass/_responsive.scss [hunk 1]
```diff
@@ -19,7 +19,7 @@
   }
 }
 
-@media #{$below_small} {
+@media (#{$below_small}) {
```
Comment (mir4a) [inline _sass/_responsive.scss]: @freevova pls remve `(#{` and `})`. For example:
before:

``` scss
@media (#{$below_small}) {
```

after:

``` scss
@media $below_small {
```

**Comment by freevova** [comment]
@EnragedWildkin I fixed bugs with foundation downloading after style change. To run it locally use 'gulp stylefix' task. Please review.

File: _sass/app.scss [hunk 2]
```diff
@@ -88,4 +88,4 @@
 @import "parts/footer";
 @import "responsive";
 @import "animation";
-@import "syntax";
+@import "syntax.scss";
```
Comment (mir4a) [inline _sass/app.scss]: @freevova pls remove `syntax.sass` and skip this change

File: _sass/custom/_ui.scss [hunk 3]
```diff
@@ -82,7 +82,8 @@ $button_min_width: 190px;
 }
 
 .button_large {
-  @extend .large-8;
+  position: relative;
```
Comment (mir4a) [inline _sass/custom/_ui.scss]: @freevova skip this changes, it should work with correct foundation import

File: Gemfile [hunk 4]
```diff
@@ -27,7 +26,8 @@ group :utils do
 end
 
 gem 'rack-contrib'
-gem 'rack-cache'
+gem 'rack-cache', github: 'rtomayko/rack-cache', branch: 'master'
```
Comment (pftg) [inline Gemfile]: why we switched to master version? pls add note about this above this line

Comment (pftg) [inline Gemfile]: by the way, rack cache is not used for github pages, i used this for heroku

**Comment by freevova** [comment]
@pftg I added path to gem 'rack-cache' because when I ran bundle I got next error "Could not find rack-cache-1.6.0 in any of the sources". Also I added master branch to use last stable version of gem.

**Comment by freevova** [comment]
@EnragedWildkin Please review.

**Comment by pftg** [comment]
@freevova have you asked 5 times Why about `rack-cahe`? pls do more research why this happened and how to fix it more freindly

File: Gemfile [hunk 5]
```diff
@@ -27,7 +26,8 @@ group :utils do
...
 gem 'dalli'
 gem 'kgio'
 gem 'puma'
+gem 'sass', '3.3.13'
```
Comment (pftg) [inline Gemfile]: why we freeze this gem ?

File: _sass/syntax.sass [hunk 6]
```diff
@@ -1,135 +0,0 @@
...
-  .bp
-    color: #999999  /* Name.Builtin.Pseudo */
-  .vc
-    color: #008080  /* Name.Variable.Class */
-  .vg
-    color: #008080  /* Name.Variable.Global */
-  .vi
-    color: #008080  /* Name.Variable.Instance */
-  .il
-    color: #009999  /* Literal.Number.Integer.Long */
```
Comment (pftg) [inline _sass/syntax.sass:135]: why this file has been removed? have you tested that site will looks good after that? pls post screenshot before and after

**Comment by pftg** [comment]
pls run gemini tests

File: _sass/syntax.sass (hunk 6 above)
Comment (mir4a) [inline _sass/syntax.sass:135]: @pftg we use identical version of this styles in `syntax.scss`. This file is redundant.

**Comment by EnragedWildkin** [comment]
@freevova any updates?

File: Gemfile (hunk 5 above)
Comment (freevova) [inline Gemfile]: There are problems with foundation importing. Here I use foundation v.5.0.3. After setting sass v.3.3.14 foundation import correctly. All tables with ruby code displayed correctly(there is no more border in the middle of table if ruby code is small).
**before:**
bad
**after:**
good

**Comment by freevova** [comment]
@mir4a @EnragedWildkin I ran gemini tests and got the following error in some tests: "_Failed to capture the element because it is positioned outside of the captured body. Most probably you are trying to capture an absolute positioned element which does not make body height to expand. To fix this place a tall enough <div> on the page to make body expand.
Element position: 0, 0; size: 1366, 5404. Page screenshot size: 1366, 5403._ ". I used the following versions: 
- gemini --version (3.0.1)
- jekyll --version (3.0.3)
- phantomjs --version (2.1.1)
  What can you advise to solve these errors?

File: _sass/syntax.sass (hunk 6 above)
Comment (pftg) [inline _sass/syntax.sass:135]: k

**Comment by freevova** [comment]
@mir4a @EnragedWildkin Updated jekyll to v '3.1.6' and github-pages to '80'. With github-pages '78' and jekyll '3.0.5' all tests fell. After updating I got 7 errors 
error
But I can't understand why. On pictures with errors there are no inconsistencies.

**Comment by mir4a** [comment]
@freevova @EnragedWildkin I suppose gemini has conflict with fractional numbers, because the height of our site is fractional:
screen shot 2016-05-31 at 12 54 36

@freevova could you please upload report of gemini tests somewhere (to firebase)? (ask @loogle18 to help with upload reports)

**Comment by freevova** [comment]
@mir4a  Uploaded report of gemini tests on firebase https://jetthoughts.firebaseapp.com/

**Comment by mir4a** [comment]
@freevova all gemini tests (except team page) fail because of animation: note swipe icon at homepage and tiny loader bar on laptop image at portfolio. Suggest to turn off animation for gemini tests. Please find out if it possible to turn off animation while gemini test.

**Comment by EnragedWildkin** [comment]
@freevova Please create new issue for that, and I will merge this branch.

**Comment by EnragedWildkin** [comment]
@freevova Please also update commit message.

**Comment by freevova** [comment]
Added new issue to turn off animation while gemini test #215

**Pull request merged**